ansible/roles/test/files/ptftests/py3/advanced-reboot.py

```diff
@@ -2180,6 +2180,7 @@ def wait_dut_to_warm_up(self):
                 up_time = None
 
             if elapsed > warm_up_timeout_secs:
+                self.log('Actual warm up time {}'.format(elapsed))
                 raise Exception("IO didn't come up within warm up timeout. Control plane: {}, Data plane: {}".format(
                     ctrlplane, dataplane))
```

Comment:
Why not add the debug info in exception message? Then when the test failed, it would be easier to find the debug message.

Reply:
Good idea.